src/integrationTest/resources/application.yaml

```diff
@@ -271,6 +285,12 @@ document:
       fl401_draft_welsh_filename: Draft_DA_Welsh_application
       fl401_c8_welsh_template: FL401-c8-welsh.docx
       fl401_c8_welsh_filename: C8_Document_Welsh.pdf
+      fl401_fl406_draft_template: PRL-GOR-ENG-FL406-Power-Of-Arrest-Draft.docx
```

Comment:
are these changes are part of your story?

Reply:
Added to ensure documents are identical to those within other test areas in the process of resolving build issues.